testsuite/integration/elytron/pom.xml

```diff
@@ -488,6 +490,7 @@
                         <jboss.inst>${wildfly.dir}</jboss.inst>
                         <module.path>${wildfly.dir}/modules</module.path>
                         <node0>${node0}</node0>
+                        <embedded.server.stability>preview</embedded.server.stability>
```

Comment:
Here, I think we can use `<embedded.stability>${embedded.server.stability}</embedded.stability>`.